dev-client/src/components/links/ExternalLink.tsx

```diff
@@ -34,18 +35,70 @@ export const ExternalLink = ({
   label,
   url,
 }: ExternalLinkProps) => {
+  const [pressed, setPressed] = useState(false);
+  const onPressIn = useCallback(() => setPressed(true), [setPressed]);
+  const onPressOut = useCallback(() => setPressed(false), [setPressed]);
+
   const isValidUrl = useMemo(() => validateUrl(url), [url]);
   const openUrl = useCallback(() => Linking.openURL(url), [url]);
 
-  return (
-    isValidUrl && (
-      <TextButton
-        type={type}
-        role="link"
-        label={label}
-        rightIcon="open-in-new"
-        onPress={openUrl}
-      />
-    )
+  const colorStyles = COLOR_STYLES[type];
+  const colorStyle = pressed ? colorStyles.pressed : colorStyles.default;
+
+  return isValidUrl ? (
```

Comment:
`ExternalLink` had to be reimplemented based on the old `TextButton` implementation, since the updated styles led to them diverging too much to make sense as a single component anymore.